Support/Testing/CircleCI/Dockerfile

```diff
@@ -51,7 +80,8 @@ RUN apt-get install -y flex bison
 
 # Install x86 compilers
 RUN apt-get install -y g++-multilib
-RUN apt-get install -y clang-6.0
+RUN apt-get install -y clang-7
+RUN ln -s clang-7 /usr/bin/clang
```

Comment:
I don't think this is needed because I don't think we ever call `clang` directly.